docs/src/HowToGuides/Numerics/DGMethods/how_to_make_a_balance_law.md

```diff
@@ -35,44 +35,44 @@ the following methods, which are computed locally at each nodal point:
+| [`vars_reverse_integrals`](@ref)         | **NO** |  specify the names of any one-dimensional vertical integral from **top to bottom** of the domain required for the balance law. each variable here must also exist in `vars_integrals` since the reverse integral kernels use subtraction to reverse the integral instead of performing a new integral. use to represent the value before **and** after reversing direction |
 
 ## Methods to compute gradients and integrals
 | **Method** |  Purpose |
 |:-----|:-----|
-| [`compute_gradient_argument!`](@ref DGMethods.compute_gradient_argument!) | specify how to compute the arguments to the gradients. can be functions of conservative state  and auxiliary variables. |
-| [`compute_gradient_flux!`](@ref DGMethods.compute_gradient_flux!) | specify how to compute gradient fluxes. can be a functions of the gradient state, the conservative state, and auxiliary variables.|
-| [`integral_load_auxiliary_state!`](@ref DGMethods.integral_load_auxiliary_state!) | specify how to compute integrands. can be functions of the conservative state and auxiliary variables. |
-| [`integral_set_auxiliary_state!`](@ref DGMethods.integral_set_auxiliary_state!) | specify which auxiliary variables are used to store the output of the integrals. |
-| [`reverse_integral_load_auxiliary_state!`](@ref DGMethods.reverse_integral_load_auxiliary_state!) | specify auxiliary variables need their integrals reversed. |
-| [`reverse_integral_set_auxiliary_state!`](@ref DGMethods.reverse_integral_set_auxiliary_state!) | specify which auxiliary variables are used to store the output of the reversed integrals. |
-| [`update_auxiliary_state!`](@ref DGMethods.update_auxiliary_state!) | perform any updates to the auxiliary variables needed at the beginning of each time-step. Can be used to solve non-linear equations, calculate integrals, and apply filters. |
-| [`update_auxiliary_state_gradient!`](@ref DGMethods.update_auxiliary_state_gradient!) | same as above, but after computing gradients and gradient fluxes in case these variables are needed during the update. |
+| [`compute_gradient_argument!`](@ref) | specify how to compute the arguments to the gradients. can be functions of conservative state  and auxiliary variables. |
+| [`compute_gradient_flux!`](@ref) | specify how to compute gradient fluxes. can be a functions of the gradient state, the conservative state, and auxiliary variables.|
+| [`integral_load_auxiliary_state!`](@ref) | specify how to compute integrands. can be functions of the conservative state and auxiliary variables. |
+| [`integral_set_auxiliary_state!`](@ref) | specify which auxiliary variables are used to store the output of the integrals. |
+| [`reverse_integral_load_auxiliary_state!`](@ref) | specify auxiliary variables need their integrals reversed. |
+| [`reverse_integral_set_auxiliary_state!`](@ref) | specify which auxiliary variables are used to store the output of the reversed integrals. |
```

Comment:
These are also pretty hard-cored tied into how the DG rhs! compute kernel works. I guess moving them here is a good first step to a proper integral interface, but these are really just interfacing with the `indefinite_stack_integral!` and `reverse_indefinite_stack_integral!` functions, which assume a DG discretization.